slsim/lens_pop.py

```diff
@@ -434,7 +126,7 @@ def get_num_sources_tested_mean(self, testarea):
         """
         num_sources = self.source_number
         num_sources_tested_mean = (testarea * num_sources) / (
-            12960000 * self._factor_source * self.source_sky_area.to_value("deg2")
+            12960000 * self._factor_source * self._sources._sky_area.to_value("deg2")
```

Comment:
no access to private variable in other classes. Make _sky_area a public variable as sources.sky_area